app/celery/scheduled_tasks.py

```diff
@@ -32,13 +32,11 @@
 from app.dao.notifications_dao import (
     dao_old_letters_with_created_status,
     dao_precompiled_letters_still_pending_virus_check,
-    is_delivery_slow_for_providers,
     letters_missing_from_sending_bucket,
     notifications_not_yet_sent,
 )
 from app.dao.provider_details_dao import (
     dao_adjust_provider_priority_back_to_resting_points,
-    dao_reduce_sms_provider_priority,
```

Comment:
This is used one other place, but perhaps can also be removed entirely?

Reply:
I'm going to take care of this one as part of #7

Reply:
Sounds good to me! (I think I missed that implication in the previous thread, sorry)

Reply:
No worries, I added that to the other thread after you had already commented on this line.